### ISSUE TITLE
document how to load specific file(s) instead of .env

### ISSUE THREAD

**Issue by keen99** [bkeepers/dotenv#266]
The syntax is straightforward, but I had to go code diving to find it since I couldn't find it in the docu:

```
   Dotenv.load(file)
```

Also - it appears that there may be a way to pass multiple files - I wasn't able to figure out the syntax for that, so I end up looping .load a few times.   if that's possible, some docu would be good - if it's not, then noting it around .load would also be good.

**Comment by nbibler** [comment]
You can just call:

```ruby
Dotenv.load('basefile', 'overriding-file1', 'overriding-file2')
```

The later files will override the previous files' settings.

**Comment by bkeepers** [comment]
Thanks for the suggestion. Fixed in 5c9d5384295d0dc41e569196ff3341f6b833bd0b.
